tests/test_inverter.rs

```diff
@@ -112,8 +112,6 @@ async fn test_replies_to_heartbeats() {
     let channels = Channels::new();
     let inverter = lxp::inverter::Inverter::new(config, &inverter, channels.clone());
 
-    let from_inverter = channels.from_inverter.subscribe();
```

Comment:
Busted :D. I'm guessing this was here to prevent the error message about closed channels from occurring.

With this change, the `Inverter::new` call on line 113 creates a subscription for us, so we don't need this line here. It resolves that clippy lint too.

Reply:
Yeah clippy has been moaning at me about this one for a while, nice to get rid of it :)